dlr/dlr-engine/src/main/java/ai/djl/dlr/engine/DlrNDManager.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.dlr.engine;
+
+import ai.djl.Device;
+import ai.djl.ndarray.BaseNDManager;
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.types.DataType;
+import ai.djl.ndarray.types.Shape;
+import java.util.Arrays;
+
+/** {@code DlrNDManager} is the DLR implementation of {@link NDManager}. */
+public class DlrNDManager extends BaseNDManager {
+
+    private static final DlrNDManager SYSTEM_MANAGER = new SystemManager();
+
+    private DlrNDManager(NDManager parent, Device device) {
+        super(parent, device);
+    }
+
+    static DlrNDManager getSystemManager() {
+        return SYSTEM_MANAGER;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public NDArray create(float[] data, Shape shape) {
```

Comment:
Can we create from Buffer? This seemed to be the most common way people would do. Also probably you need to override the other creation method to avoid calling from it

Reply:
I was thinking about that as well. After I searched [Buffer vs primitive array](https://stackoverflow.com/questions/18913001/when-to-use-array-buffer-or-direct-buffer). array seem to be a better choice cause we eventually will pass float[] to JNI not the data pointer or byte array

Reply:
I think we should use Buffer, if it's DirectBuffer, we don't even need copy in JNI.